libs/application/templates/parental-leave/src/fields/Rights/GiveDaysSlider.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { FC, useState } from 'react'
+import { FieldBaseProps, getValueViaPath } from '@island.is/application/core'
+import { Controller, useFormContext } from 'react-hook-form'
+import { Box, Text } from '@island.is/island-ui/core'
+import { theme } from '@island.is/island-ui/theme'
+import { useLocale } from '@island.is/localization'
+import { m } from '../../lib/messages'
+import Slider from '../components/Slider'
+import BoxChart, { BoxChartKey } from '../components/BoxChart'
+import config from '../../config'
+
+const GiveDaysSlider: FC<FieldBaseProps> = ({ field, application }) => {
+  const maxDays = config.maxDaysToGiveOrReceive
+  const { id } = field
+  const { formatMessage } = useLocale()
+  const currentAnswer = getValueViaPath(
+    application.answers,
+    field.id,
+    1,
+  ) as number
+
+  const { clearErrors } = useFormContext()
+
+  const [chosenGiveDays, setChosenGiveDays] = useState<number>(currentAnswer)
+
+  const daysStringKey = chosenGiveDays > 1 ? m.giveRightsDays : m.giveRightsDay
+
+  const yourRightsWithGivenDaysStringKey =
+    maxDays - chosenGiveDays === 1
+      ? m.yourRightsInMonthsAndDay
+      : m.yourRightsInMonthsAndDays
+
+  const boxChartKeys: BoxChartKey[] = [
+    {
+      label: () => ({
+        ...yourRightsWithGivenDaysStringKey,
+        values: { months: '5', day: maxDays - chosenGiveDays },
```

Comment:
Moving this to config? or simply doing a subtract here, however, might be better to keep it referenced in the config if it is related to the "law" number e.g. `minMonths`

Reply:
Good point, fixed now as `minMonths`.